### ISSUE TITLE
test_organic_flow_param_values is failing; need to use "(none)" as medium, instead of "referral"

### ISSUE THREAD

**Issue by stephendonner** [mozilla/stubattribution-tests#158]
stubattribution.stage[0] is failing in one test, ```test_organic_flow_param_values```, using ```PYTEST_BASE_URL=https://www.allizom.org```:

```
22:17:01 [gw2] [100%] FAILED tests/test_organic_flow.py::test_organic_flow_param_values 
22:17:01 
22:17:01 =================================== FAILURES ===================================
22:17:01 ________________________ test_organic_flow_param_values ________________________
22:17:01 [gw2] linux2 -- Python 2.7.13 /usr/local/bin/python
22:17:01 tests/test_organic_flow.py:47: in test_organic_flow_param_values
22:17:01     assert actual == expected
22:17:01 E   AssertionError: assert {'campaign': ....mozilla.org'} == {'campaign': '....allizom.org'}
22:17:01 E     Common items:
22:17:01 E     {'campaign': '(not set)', 'content': '(not set)'}
22:17:01 E     Differing items:
22:17:01 E     {'source': 'www.mozilla.org'} != {'source': 'www.allizom.org'}
22:17:01 E     {'medium': '(none)'} != {'medium': 'referral'}
22:17:01 E     Full diff:
22:17:01 E     {'campaign': '(not set)',
22:17:01 E     'content': '(not set)',
22:17:01 E     -  'medium': '(none)',
22:17:01 E     +  'medium': 'referral',
22:17:01 E     -  'source': 'www.mozilla.org'}
22:17:01 E     ?                 ^^^^  ^
22:17:01 E     +  'source': 'www.allizom.org'}
22:17:01 E     ?                 ^  ^^^^
22:17:01 ------------------------------- pytest-selenium --------------------------------
22:17:01 URL: https://www.allizom.org/en-US/firefox/download/thanks/
22:17:01 Sauce Labs Job: http://saucelabs.com/jobs/5f8ed4c0e71e4d3586350c69215f7f5a
22:17:01  generated xml file: /home/jenkins/slave/workspace/stubattribution.stage@2/results/junit.xml 
22:17:01  generated html file: /home/jenkins/slave/workspace/stubattribution.stage@2/results/index.html 
22:17:01 =========================== short test summary info ============================
22:17:01 FAIL tests/test_organic_flow.py::test_organic_flow_param_values
22:17:01 ===================== 1 failed, 2 passed in 22.26 seconds ======================
```

[0] https://qa-master.fxtest.jenkins.stage.mozaws.net/job/stubattribution.stage/4415/

**Comment by stephendonner** [comment]
I looked into this, but still could use some help with verifying my approach:

we don't send Chrome to https://www.mozilla.org/en-US/firefox/new/ in the organic scenario - at least not with my test code/flow
we send them straight to https://www.mozilla.org/firefox/download/thanks/
so we're missing https://github.com/mozilla/stubattribution-tests/blob/217a01782d04d73d6cbdc847402785615c2d4966/pages/download.py#L7 which is only on /firefox/new
guess I just need to update locators to use the /firefox/ page, rather than the one on /firefox/new but I want to be sure

**Comment by stephendonner** [comment]
Digging in further, and testing manually:

1) it's *not* locators; we're always getting the stubattribution, it just differs in its attribute values
2) we currently expect and set ```"referral"``` for ```attribution.medium```, here: https://github.com/mozilla/stubattribution-tests/blob/217a01782d04d73d6cbdc847402785615c2d4966/tests/test_organic_flow.py#L42
3) in a manual test with Chrome, both ```attribution.campaign``` and ```attribution.content``` are ```"(not set)"```, whereas ```attribution.medium``` is now ```"(none)"```

**Comment by davehunt** [comment]
Fixed by #159
